docs/framework/wpf/advanced/how-to-create-outlined-text.md

```diff
@@ -51,5 +51,7 @@ Example of an image brush applied to the stroke and highlight
  [!code-csharp[OutlineTextControlViewer#OnRender](../../../../samples/snippets/csharp/VS_Snippets_Wpf/OutlineTextControlViewer/CSharp/OutlineTextControl.cs#onrender)]
  [!code-vb[OutlineTextControlViewer#OnRender](../../../../samples/snippets/visualbasic/VS_Snippets_Wpf/OutlineTextControlViewer/visualbasic/outlinetextcontrol.vb#onrender)]  
   
+  For the source of the example custom user control object, see ![OutlineTextControl](../../../../samples/snippets/csharp/VS_Snippets_Wpf/OutlineTextControlViewer/CSharp/OutlineTextControl.cs)
```

Comment:
This is interpreted as a link to content in the doc set and can't be resolved. This has to be a link to the GitHub repo: \[OutlineTextControl.cs](https://github.com/dotnet/samples/blob/master/snippets/csharp/VS_Snippets_Wpf/OutlineTextControlViewer/CSharp/OutlineTextControl.cs)

Reply:
I think using the absolute path won't work in this case because of how the samples reference in OPS works.

Reply:
So from what I'm seeing then it's just not feasible unless we wanted to stick the entire example into a snippet or something of the sort. I'll subscribe to this PR and Issue #7818 and put an issue in with the docs feedback repo.

Reply:
Sorry, I didn't realize you were trying to create a regular link here. So, in this case, we should probably link to both C# and VB versions.